package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1098,7 +1098,7 @@ def omega_selections(residues, c_name='C', n_name='N', ca_name='CA'):
     transplants[ResidueGroup].append(('omega_selections', omega_selections))
 
     def chi1_selection(residue, n_name='N', ca_name='CA', cb_name='CB',
-                       cg_name='CG'):
+                       cg_name='CG CG1 OG OG1 SG'):
         """Select AtomGroup corresponding to the chi1 sidechain dihedral N-CA-CB-CG.
```

Comment:
Given your research into chi1 definitions, could you _please_ record your reasoning here and perhaps write the chi1 as N-CA-CB-*G where *G = is the heavy atom in the gamma position?

The Janin 1978 paper https://doi.org/10.1016/0022-2836(78)90408-4 also talks about chi1 but I didn't have time to find the exact definition. Something about IUPAC 1970...(?)

Reply:
Which reminds me: our Janin analysis and the chi1 selection here need to be consistent.

Reply:
We will need to decide if we want to keep to the stricter definition where the gamma can only be "C", then (I am leaning... yes). I don't think the IUPAC 1970 document actually specifies that the gamma atom must be carbon, but a number of secondary sources citing IUPAC 1970 have rolled with that definition (and for Janin particularly, because of energy differences). The Janin class ignores residues without a carbon in that gamma position.

Janin 1978:

> Our calculations apply to residues with carbon atoms in the y-position. Sulphur (Cys) and oxygen atoms (Ser, Thr) have lesser effects (Tonnuswamy & Sasisekharan, 1971a). For side-chains with branched Cb atoms (Ile, Val, Thr) the energy barriers created by the two y-substituents add up

IUPAC-IUB Commission 1969 Draft
> 4.5.2 Torsion angles are denoted by x and are specified by two (or three) superscripts, the first one (or two), in the situation
> ```
>						D
>						|
> 				A-B-C--E
>						|
>						F
> ```
> indicating the bond B-C about which the angle is measured, and the last indicating whether the angle is measured relative to D, E, or F. The principal torsion angle is defined by Rule 2.2.1, and if there is no ambiguity the last superscript may be omitted in referring to it.
> Thus in valine $\chi_i^{2, 1}$ refer to the torsion angles specifying $C_i^{y1}$ and $C_i^{y2}$ [...]

^^ so IUPAC reckons we need to be specific with which chi1 angle to use, and Janin ignores branched cases and non-Carbon cases.

Reply:
Thanks for doing the research. I didn't realize that Janin does not do "all" chi1.

Given that branched amino acids have more than a chi1, can we consistently use one and state that? (Then we could later update our Janin analysis to also collect those... or at least be clear that we follow the original paper and exclude branched aa.)

I feel that when using chi1 you really want to know what the sidechain does and not getting any answer for Thr or Cys feels odd.

Reply:
I added a warning that the Janin class differs from the `chi1_selection()` function. Turns out the Janin class already has a note saying that by default these certain residues are ignored, although it attributes them all to not having a chi2 angle (which true, they do not), not because the paper defines it as carbon-only.